Mathlib/Algebra/Order/GroupWithZero/Range.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2025 Filippo A. E. Nuccio. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Filippo A. E. Nuccio
+-/
+import Mathlib.Algebra.GroupWithZero.Range
+import Mathlib.Algebra.Order.GroupWithZero.WithZero
+
+/-! # The range of a MonoidWithZeroHom
+
+Given a `MonoidWithZeroHom` `f : A → B` whose codomain `B` is a `LinearOrderedCommGroupWithZero`,
+we provide some order properties of the `MonoidWithZeroHom.valueGroup₀` as defined in
+`Mathlib.Algebra.GroupWithZero.Range`.
+
+-/
+
+namespace MonoidWithZeroHom
+
+variable {A B : Type*}
+variable [MonoidWithZero A] [LinearOrderedCommGroupWithZero B]
+variable {f : A →*₀ B}
+
+open WithZero
+
+/-- The inclusion of `valueGroup₀ f` into `WithZero Bˣ` as an homomorphism of monoids with zero. -/
+@[simps!]
+def valueGroup₀.monoidWithZeroHom : ValueGroup₀ f →*₀ WithZero Bˣ :=
```

Comment:
Do you really need this definition? I thought the correct one was `embedding` above